tests/integration/tests/01_install_ibm_installation_manager_from_http.rb

```diff
@@ -12,10 +12,16 @@
   end
 end
 
+if agent['platform'] =~ /aix/
+  source = 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.aix.gtk.ppc_1.8.4000.20151125_0201.zip'
+elsif agent['platform'] =~ /linux/
+  source = 'http://int-resources.ops.puppetlabs.net/QA_resources/ibm_websphere/agent.installer.linux.gtk.x86_64_1.8.3000.20150606_0047.zip'
+end
```

Comment:
This block of code is repeated in different test cases. You should move this into a helper method to reduce code maintenance in the future.